static/js/load_schedule.js

```diff
@@ -1,180 +1,233 @@
+    $('footer').css('display','none');
+    $('div#user_pref').css('display','none');
+    $('div#news').css('display','none');
+    $('img#qrcode1').css('display','none');
+    $('img#qrcode2').css('display','none');
 }
 
-$(window).on('load',function(){
-    window.setInterval(timer,1000);
-});
+function setDateInfo(jsonRes){
+    var datetime = new Date(Date.parse(jsonRes.file_text.date));
+    var weekday = getWeekday(datetime);
+    var year = datetime.getFullYear();
+    var month = datetime.getMonth() + 1;
+    var date = datetime.getDate();
+    $('div#week b').text(weekday);
+    $('div#year b').text(year);
+    $('div#date b').text(String(month)+'.'+String(date));
+}
+
+function getWeekday(datetime){
+    var week = ["Mon.", "Tue.", "Wed.", "Thu.", "Fri.", "Sat.", "Sun."];
+    return week[datetime.getDay()];
+}
+
+function setPreferenceDisplay(){
+    $('div#user_pref').css('display','inline');
+    $('img#qrcode1').css('display','inline');
+    $('img#qrcode2').css('display','inline');
+}
+
+function setConstellationInfo(jsonRes){
+    $('div#name b').text(jsonRes.file_text.nickname);
+    $('div#star b').text(jsonRes.file_text.constellation.name);
+    $('div#overall b').text(jsonRes.file_text.constellation.value[0]);
+    $('div#love b').text(jsonRes.file_text.constellation.value[1]);
+    $('div#career b').text(jsonRes.file_text.constellation.value[2]);
+    $('div#wealth b').text(jsonRes.file_text.constellation.value[3]);
+}
+
+function setNewsDisplay(){
+    $('div#news').css('display','inline');
+}
+
+function setNewsInfo(jsonRes){
+    $('div#forum-1').text(jsonRes.file_text.forum_name1);
+    $('p#title-1').text(jsonRes.file_text.title1);
+    $('img#qrcode-1').attr('src',String(jsonRes.file_text.QR1));
+    $('div#forum-2').text(jsonRes.file_text.forum_name2);
+    $('p#title-2').text(jsonRes.file_text.title2);
+    $('img#qrcode-2').attr('src',String(jsonRes.file_text.QR2));
+}
+
+function setAnnouncementDisplay(jsonRes){
+    $('div.title2').css('display','inline');
+    if(jsonRes.type_name === "獲獎公告"){
+        $('footer').css('display','inline');
+    }
+    else{
+        $('footer').css('display','none')
+    }
+}
+
+function setConInfo(jsonRes){
+    if ( 'con' in jsonRes.file_text){
+        $('span#con').html(jsonRes.file_text.con);
+    }
+    else{
+        if (jsonRes.type_name === "獲獎公告"){
+            $('span#con').html("賀！");
+        }
+        else{
+            $('span#con').html("活動公告");
+        }
+    }
+}
+
+function setAnnouncementInfo(jsonRes){
+    $('#title1').html(jsonRes.file_text.title1);
+    $('#title2').html(jsonRes.file_text.title2);
+    if(jsonRes.file_text.location != ""){
+        $('#location').html("地點： " + jsonRes.file_text.location)
+    }
+    else{
+        $('#location').html("")
+    }
+    if(jsonRes.file_text.detailtime != ""){
+        $('#detailtime').html("時間： " + jsonRes.file_text.detailtime)
+    }
+    else{
+        $('#detailtime').html("")
+    }
```

Comment:
When this commit merge back to master. Will the conflict appear? Or the time & location bug will be back?

Reply:
@stvreumi I think you will be more familiar with this part than me.

Reply:
I try to switch the branch to see what will happen. Fortunately, the conflict will appear.
Although, the function part, which cause the bug reborn wont be conflict part. 
We can avoid the bug come back if we remember to modify the function part when solving the conflict.

Reply:
@icecat2012 Did you mean that the bug won't appear again when dev branch merge into master branch?

Reply:
The bug will appear again when merging back to master.
However, there will got a conflict when trying merge back to master.
So we can debug when solving the conflict.

Reply:
Ok, got it.

Reply:
So, should I modify the code to solve the existing bug or...?

Reply:
I update the code from the closed PR.
I believe it should fix the bug.